packages/live-share-media/src/VolumeLimiter.ts

```diff
@@ -3,59 +3,64 @@
  * Licensed under the Microsoft Live Share SDK License.
  */
 
- import { TimeInterval } from '@microsoft/live-share';
- import { IMediaPlayer } from './IMediaPlayer';
+import { TimeInterval } from '@microsoft/live-share';
+import { IMediaPlayer } from './IMediaPlayer';
 
 export enum LevelType { fixed, percentage }
 
+/**
+ * Smooth audio level changes when selectedVolume is modified, or if volume limiting has started/ended.
+ */
 export class VolumeLimiter {
     private readonly _player: IMediaPlayer;
+    private readonly _rampDuration = new TimeInterval(500);
```

Comment:
Is the policy that we don't require documentation for private content? I think it would be good to document these fields.

If I didn't have the diff for reference, I wouldn't know what is meant by `rampDuration`